dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBNoSQL/AzureCosmosDBNoSQLMemoryStore.cs

```diff
@@ -18,11 +17,13 @@
 
 namespace Microsoft.SemanticKernel.Connectors.AzureCosmosDBNoSQL;
 
+#pragma warning disable SKEXP0001 // IMemoryStore is experimental (but we're obsoleting)
+
 /// <summary>
 /// An implementation of <see cref="IMemoryStore"/> backed by a Azure Cosmos DB database.
 /// Get more details about Azure Cosmos DB vector search  https://learn.microsoft.com/en-us/azure/cosmos-db/
 /// </summary>
-[Experimental("SKEXP0020")]
+[Obsolete("The IMemoryStore abstraction is being obsoleted, use Microsoft.Extensions.VectorData and AzureCosmosDBNoSQLMemoryStore")]
```

Comment:
Same in other places in Azure CosmosDB NoSQL connector.
```suggestion
[Obsolete("The IMemoryStore abstraction is being obsoleted, use Microsoft.Extensions.VectorData and AzureCosmosDBNoSQLVectorStore")]
```